code/game/objects/items/weapons/chaplain_items.dm

```diff
@@ -67,11 +68,59 @@
 /obj/item/nullrod/charm/get_mask_examine_text(mob/user)
 	return "around [user.get_pronoun("his")] neck"
 
-/obj/item/nullrod/obsidianshards
-	name = "obsidian shards"
-	desc = "A loose pile of obsidian shards, waiting to be assembled into a religious focus."
-	icon_state = "nullshards"
-	item_state = "nullshards"
+/obj/item/nullrod/matake
+	name = "\improper Mata'ke sword"
+	desc = "A ceremonial spear crafted after the image of Mata'ke's holy weapon."
+	icon_state = "matake_spear"
+	item_state = "matake_spear"
+	slot_flags = SLOT_BELT | SLOT_BACK
+	w_class = ITEMSIZE_LARGE
+
+/obj/item/nullrod/rredouane
+	name = "\improper Rredouane sword"
+	desc = "A ceremonial sword crafted after the image of Rredouane's holy sword."
+	icon_state = "rredouane_sword"
+	item_state = "rredouane_sword"
+
+/obj/item/nullrod/shumaila
+	name = "\improper Shumaila hammer"
+	desc = "A ceremonial hammer carried by the priesthood of Shumaila."
+	icon_state = "shumaila_hammer"
+	item_state = "shumaila_hammer"
+
+/obj/item/nullrod/zhukamir
+	name = "\improper Zhukamir ladle"
+	desc = "A golden ladle used by Zhukamir's most faithful worshippers."
+	icon_state = "zhukamir_ladle"
+	item_state = "zhukamir_ladle"
+
+/obj/item/nullrod/azubarre
```

Comment:
It may be a good idea to have `isFlameSource()` return lit for this item, so it can light stuff on fire.